### ISSUE TITLE
[Suggestion] Add Missing Blaze Powder to the `c:dusts` tag

### ISSUE THREAD

**Issue by ffuentesm** [neoforged/NeoForge#1257]
Compared to the previous version, the Blaze Dust is missing from the Dust Tag list. 

Just add this file to the src/generated/resources/data/c/tags/item/dusts

```
{
  "values": [
    "minecraft:blaze_powder"
  ]
}
```
As well as change the dusts.json file in src/generated/resources/data/c/tags/item/
```
{
  "values": [
    "#c:dusts/glowstone",
    "#c:dusts/redstone",
    "#c:dusts/blaze",
    {
      "id": "#forge:dusts",
      "required": false
    }
  ]
}
```

**Comment by TelepathicGrunt** [comment]
I'm on the fence about this one. I would like to hear from more people on this. Does it make sense to have in the tag? And it won't interfere with people currently consuming the dust tag right?

**Comment by ApexModder** [comment]
I agree that `blaze_powder` should be added to the `dusts` tag, its the dust variant of `blaze` and tbh most mods already treat it as such from what ive seen.

Pulverizers/Crushers crush `blaze` into `blaze_powder`
<sub>(vanilla also does this via shapless crafting)</sub>

**Comment by TelepathicGrunt** [comment]
From fabric post, someone said:
> If blaze powder is a dust, then presumably gunpowder is a dust as well.

**Comment by TelepathicGrunt** [comment]
I believe the origin/use of the dust tag beyond just redstone dust and glowstone dust was to collect modded machines crushing ingots into dust form and then sticking it into the tag to group it up. Just as background info

**Comment by KnightMiner** [comment]
Yeah, in my mods the only time I really use `dusts` as a hierarchy is to know that `dusts/<metal>` can be smelted into `ingots/<metal>` making its value exactly 1 ingot (no ore multiplication).

Glowstone and redstone don't fit that (anyone adding `ingots/redstone` means something very different, usually they go with something like `ingots/redstone_alloy`). Redstone could just as easily be `c:redstone` or `c:powders/redstone` and it wouldn't impact the usages. So blaze powder being a dust or not has no impact on me; are people adding alternative types of blaze powder?

**Comment by TelepathicGrunt** [comment]
So there’s:

Gunpowder
Redstone Dust
Glowstone Dust
Blaze Powder
Concrete Powder
Powder Snow (Powder Snow Bucket)
Sugar

If someone can make a list of which ones should go in the dust item tag and what the javadoc should say to make it clear why just those items are in the tag, then I think we can make progress.

**Comment by KnightMiner** [comment]
Any "dust" that is a full block certainly does not belong there, or you have to add sand and gravel. At that point the tag lost all meaning and is just a philosophical categorization.
